File: C/Vetor/SomaVetores.c
#include<stdio.h>
int main(){

    int i, n;

    printf("Quantos valores vai ter cada vetor?");
    scanf("%d", &n);
            int vetA[n], vetB[n], vetC[n];

        printf("Digite os valores do vetor A:\n");
            for(i=0; i<n; i++){
                scanf("%d", &vetA[i]);
    }
            printf("Digite os valores do vetor B:\n");
                for(i=0; i<n; i++){
                    scanf("%d", &vetB[i]);
    }
                printf("VETOR RESULTANTE: \n");
                    for(i=0; i<n; i++){
                        vetC[i] = vetA[i] + vetB[i];
                            printf("%d\n", vetC[i]);
                    }

}

File: Phyton/Vetor/somaVetores.py
n = int(input("Quantos valores vai ter cada vetor? "))

vetA = [0 for x in range(n)]
vetB = [0 for x in range(n)]
vetC = [0 for x in range(n)]

print("Digite os valores do vetor A: ")

for i in range(n):
    vetA[i] = int(input())

print("Digite os valores do vetor B: ")

for i in range(n):
    vetB[i] = int(input())

print("VETOR RESULTANTE: ")
for i  in range(n):
    vetC[i] = vetA[i] + vetB[i]
    print(vetC[i])


